src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -3836,6 +3839,7 @@ void ExportMusicXml::chord(Chord* chord, staff_idx_t staff, const std::vector<Ly
             QString rest = slurTieLineStyle(tieFor);
             if (tieFor->slurDirection() != DirectionV::AUTO && configuration()->musicxmlExportLayout()) {
```

Comment:
```tieFor->slurDirection() != DirectionV::AUTO```
I vote against this condition=)
1. People already [asked this earlier](https://musescore.org/en/node/21435) and Musescore team [fixed it a few years ago](https://github.com/musescore/MuseScore/pull/4488/commits/b0cc6b1609d7fd7bdc0d0cbf7123028f76d12b39)
2. By using export configuration layout you are already allowing the user to opt out of this option
3. As mentioned above: let the importing application decide what to use and what to ignore

and thank you very much @Jojo-Schmitz for your effort.

Reply:
Well, yes, you have a point there. Initially I wanted to export them only when not customized (but then always), then the condition reg. preferences came on top.

OTH that preference setting should apply for many more elements. Not sure whoe (or whether) to deal with those.

Reply:
Hmm, how about `if (tieFor->slurDirection() != DirectionV::AUTO ||  configuration()->musicxmlExportLayout())`?

Reply:
I think there is a little inconsistency here. For example, Musescore user exports without layout option, but still gets placement for some slurs/ties he changed. 
Anyway it's better than writing placement for manually positioned slurs/ties only.

Reply:
That is the case for all other placements too

Reply:
Guess we'd need input on how to deal with this from @lvinken and/or @cbjeukendrup ?

Reply:
OK, let's go for that then